charts/settings/values.yaml

```
@@ -4,6 +4,17 @@ global:
       repository: registry.gitlab.com/gitlab-org/build/cng/kubectl
       tag: v17.2.4
 
+botAccounts: {}
+  # - username: eric110
```

```suggestion
  # - username: renovatebot
```
may be better to have a more representative example here